### PR TITLE
Fix contact picker and toggle. [#725, #742]

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: components/contactPicker/index.html [hunk 1]
```diff
@@ -10,9 +10,8 @@
             <div class="letter-header">{{ $key }}</div>
 
             <div v-repeat="person : $value" class="contact">
-                <label class="circle-toggle" v-class="on: person.selected">
+                <label class="circle-toggle" v-on="click: person.selected = !person.selected" v-class="on: person.selected">
                     <span class="fa fa-check"></span>
-                    <input type="checkbox" v-model="person.selected">
```
Comment (k88hudson) [inline components/contactPicker/index.html]: unfortunately checkboxes surrounded by labels are really strange/firing 2 click events.

File: lib/index.js [hunk 2]
```diff
@@ -103,6 +103,6 @@ var app = new Vue({
         });
     },
     ready: function () {
-        attachFastClick(document.body);
+        //attachFastClick(document.body);
```
Comment (k88hudson) [inline lib/index.js]: this is causing issues. will have to revisit later

File: views/share/index.html [hunk 3]
```diff
@@ -39,4 +39,4 @@
 
 <div v-component="tabBar"></div>
 
-<div v-class="active: showPicker" v-component="contactPicker" v-with="show: showAndroid, contacts: contacts, modeledContacts: modeledContacts"></div>
+<div v-class="active: showPicker" v-component="contactPicker" v-with="show: showPicker, contacts: contacts, modeledContacts: modeledContacts"></div>
```
Comment (k88hudson) [inline views/share/index.html]: my bad on this one!

**Comment by k88hudson** [comment]
R @thisandagain

File: components/contactPicker/index.html (hunk 1 above)
Comment (thisandagain) [inline components/contactPicker/index.html]: Weird. Android issue or do we have some errant event handler?

File: components/toggle/index.js [hunk 4]
```diff
@@ -2,6 +2,14 @@ module.exports = {
     className: 'toggle',
     template: require('./index.html'),
     paramAttributes: ['disabled', 'checked'],
+    methods: {
+        onClick: function (e) {
+            e.preventDefault();
+            if (this.disabled) return;
+            this.checked = !this.checked;
+            console.log(this.checked);
```
Comment (thisandagain) [inline components/toggle/index.js]: Leftover `console.log`

File: components/contactPicker/index.html (hunk 1 above)
Comment (k88hudson) [inline components/contactPicker/index.html]: I think it's an android issue. I isolated it and still seemed to have issues

**Comment by thisandagain** [comment]
Stray debug statement. Otherwise R+!

**Pull request merged**